### PR TITLE
Stop never-ending indeterminate progress bar

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
Stop never-ending indeterminate progress bar.

As long as there is an uncleared item, created by the
alertService.start, the progress bar will keep going.

**Issue**
#3975

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

File: src/app/workflow/permissions/permissions.component.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ export class PermissionsComponent implements OnInit {
         .addWorkflowPermission(this.workflow.full_workflow_path, { email: value, role: permission }, this.entryType === EntryType.Service)
         .subscribe(
           (userPermissions: Permission[]) => {
+            this.alertService.detailedSuccess('Permissions updated');
```
Comment (svonworl) [inline src/app/workflow/permissions/permissions.component.ts:79]: Should we add similar calls to the alert service in the `remove()` method, also?

**Review by denis-yuen — COMMENTED** [review]
build issue seems relevant (error comes from application code)

File: src/app/workflow/permissions/permissions.component.ts (hunk 1 above)
Comment (coverbeck) [reply src/app/workflow/permissions/permissions.component.ts:81]: Added

**Comment by coverbeck** [comment]
> build issue seems relevant
> 
> > TypeError: The following error originated from your application code, not from Cypress.
> 
> > Cannot read properties of undefined (reading 'nativeElement')
> 
> > When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

Seems flaky, alas; it was marked so in CircleCI. Rebased and now it's passing.

File: src/app/workflow/permissions/permissions.component.ts [hunk 2]
```diff
@@ -76,6 +78,7 @@ export class PermissionsComponent implements OnInit {
...
```
Comment (denis-yuen) [inline src/app/workflow/permissions/permissions.component.ts:81]: out of a silly desire for consistency, adding permissions results in "Permissions updated" but removing permissions actually says that permissions were removed :wink:

**Review by denis-yuen — APPROVED** [review]

File: src/app/workflow/permissions/permissions.component.ts (hunk 2 above)
Comment (coverbeck) [reply src/app/workflow/permissions/permissions.component.ts:81]: I like consistency. :)

**Review by denis-yuen — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**